main.py

```diff
@@ -1,10 +1,59 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race_info: dict) -> int:
+    race, created = Race.objects.get_or_create(
+        name=race_info["name"],
+        defaults={"description": race_info["description"]},
+    )
+
+    return race.id
+
+
+def create_skill(skill_info: dict, race_id: int) -> None:
+    if not Skill.objects.filter(name=skill_info["name"]).exists():
+        Skill.objects.create(
+            name=skill_info["name"], bonus=skill_info["bonus"], race_id=race_id
+        )
```

Comment:
Read about django [get_or_create](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create) method and try to use it